JAICore/jaicore-search/src/test/java/jaicore/search/testproblems/npuzzle/parentDiscarding/PDPuzzleGenerator.java

```diff
@@ -109,38 +114,38 @@ public PDPuzzleNode move(PDPuzzleNode n, String move) {
 	 * 		The movement on the y-axis. This value should be -1 if going left, 1 if going right.
 	 * 		Otherwise it should be 0.
 	 */
-	public PDPuzzleNode move(PDPuzzleNode n,int y, int x) {
+	public PDPuzzleNode move(final PDPuzzleNode n,final int y, final int x) {
 		//cloning the board for the new node
-		
+
 		if(x == y || Math.abs(x)>1 || Math.abs(y)>1) {
 			System.out.println("No valid move. No move is executed");
 			return null;
 		}
-		
-		int[][] b = new int[dimension][dimension];
+
+		int[][] b = new int[this.dimension][this.dimension];
 		int[][] board=n.getBoard();
-		for(int i = 0; i< dimension; i++) {
-			for(int j= 0; j < dimension ; j++) {
+		for(int i = 0; i< this.dimension; i++) {
+			for(int j= 0; j < this.dimension ; j++) {
 				b[i][j] = board[i][j];
 			}
 		}
 		int eX = n.getEmptyX();
 		int eY = n.getEmptyY();
-//		int help = b[eY][eX];
+		//		int help = b[eY][eX];
 		b[eY][eX] = b[eY +y][eX+x];
 		b[eY+y][eX+x] = 0;
-		
-		PDPuzzleNode node = new PDPuzzleNode(b, eX+x, eY+y, n.getNumberOfMoves()+1);
-		
-		return node;		
+
+		PDPuzzleNode node = new PDPuzzleNode(b, eX+x, eY+y);
+
+		return node;
 	}
 
 
 
 	@Override
-	public void setNodeNumbering(boolean nodenumbering) {
+	public void setNodeNumbering(final boolean nodenumbering) {
 		// TODO Auto-generated method stub
```

Comment:
Either write a comment that this method is left blank intentionally or implement this method stub

Reply:
done